src/openssl.c

```diff
@@ -10270,7 +10310,6 @@ static int sx__gc(lua_State *L) {
 	return 0;
 } /* sx__gc() */
 
-
```

Comment:
The style of this file has 2 new lines between sections

Reply:
Should have been fixed.